.pre-commit-config.yaml

```diff
@@ -0,0 +1,8 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 22.12.0
+    hooks:
+      - id: black
+        language_version: python3.12
+
+
```

Comment:
Why do we only add black and not switch to a more powerful pre-commit framework as [ruff]( https://github.com/astral-sh/ruff ) ? (To be sync with MeshPy ?)

Reply:
This is the same pre-commit file as is used in MeshPy. A switch to `ruff` is likely happening in the near future, then we should keep the two repos in sync.